ethclient/ethclient.go

```diff
@@ -708,31 +632,19 @@ func (ec *client) EstimateGas(ctx context.Context, msg ethereum.CallMsg) (uint64
 	return uint64(hex), nil
 }
 
-// EstimateBaseFee tries to estimate the base fee for the next block if it were created
-// immediately. There is no guarantee that this will be the base fee used in the next block
-// or that the next base fee will be higher or lower than the returned value.
-func (ec *client) EstimateBaseFee(ctx context.Context) (*big.Int, error) {
-	var hex hexutil.Big
-	err := ec.c.CallContext(ctx, &hex, "eth_baseFee")
-	if err != nil {
-		return nil, err
-	}
-	return (*big.Int)(&hex), nil
-}
-
 // SendTransaction injects a signed transaction into the pending pool for execution.
 //
 // If the transaction was a contract creation use the TransactionReceipt method to get the
 // contract address after the transaction has been mined.
-func (ec *client) SendTransaction(ctx context.Context, tx *types.Transaction) error {
+func (ec *Client) SendTransaction(ctx context.Context, tx *types.Transaction) error {
 	data, err := tx.MarshalBinary()
 	if err != nil {
 		return err
 	}
 	return ec.c.CallContext(ctx, nil, "eth_sendRawTransaction", hexutil.Encode(data))
 }
 
-func ToBlockNumArg(number *big.Int) string {
+func toBlockNumArg(number *big.Int) string {
```

Comment:
I slightly think it's an easier migration to have a diff that only exports the function rather than having a copied code.